test/integration/apiserver/admissionwebhook/admission_test.go

```diff
@@ -594,8 +594,8 @@ func testWebhookAdmission(t *testing.T, watchCache bool) {
 			convertedResources[gvr.Resource] = gvr
 			// add an admission rule indicating we can receive this version
 			convertedV1beta1Rules = append(convertedV1beta1Rules, admissionregistrationv1beta1.RuleWithOperations{
-				Operations: []admissionregistrationv1beta1.OperationType{admissionregistrationv1beta1.OperationAll},
-				Rule:       admissionregistrationv1beta1.Rule{APIGroups: []string{gvr.Group}, APIVersions: []string{gvr.Version}, Resources: []string{gvr.Resource}},
+				Operations: []admissionregistrationv1.OperationType{admissionregistrationv1.OperationAll},
```

Comment:
would type aliasing {v1alpha1,v1beta1}.OperationType to admissionregistrationv1.OperationType let us avoid needing to change any references like this?

Reply:
This would not help since OperationType is inside RuleWithOperations. I have removed OperationType from v1alpha1 and v1beta1. 😅

Reply:
> This would not help since OperationType is inside RuleWithOperations.

are you sure? I thought the alias would let you use it in either direction

> I have removed OperationType from v1alpha1 and v1beta1. 😅

we can't drop the v1beta1 types/constants

Reply:
v1beta1.RuleWithOperations alias from v1.RuleWithOperations and OperationType is used in v1.RuleWithOperations.Operations. We shouldn't drop even v1beta1.OperationType is not used anywhere?

Reply:
Thanks for the info! I have updated the PR accordingly :)